src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working.");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. For example, 'Truck has stopped working.' would be more descriptive and in line with the checklist requirements.